payjoin-cli/Cargo.toml

```diff
@@ -28,7 +28,7 @@ v2 = ["payjoin/v2", "payjoin/io"]
 [dependencies]
 anyhow = "1.0.70"
 async-trait = "0.1"
-bip21 = "0.5.0"
+bip21 = { git = "https://github.com/Kixunil/bip21.git", rev = "refs/pull/26/head" }
```

Comment:
@spacebear21 any opposition to merging the git head to resolve this issue? This would block our release of 0.21.0 until bip21-0.6.0 gets released. We could still open release PRs until either 0.6.0 gets released or we release our own fork.

I think we can recognize that this solution works by merging it, and we'll have an implicit block on release until it's resolved.

Reply:
I'm not opposed to merging it with the git repo dependency, but as it stands currently the commit in Cargo.lock is out of sync with the latest https://github.com/Kixunil/bip21/pull/26. Perhaps we should specify the exact commit SHA-1 in `rev` instead of a reference to head which might change later? At a minimum we should regenerate Cargo.lock before merging.

Reply:
That sounds right to me. I just opened a this PR https://github.com/payjoin/rust-payjoin/pull/379 with a script for updating lock files